### PR TITLE
Improved css for index, users and feed

### LINKED ISSUE #24: Fix CSS
**Describe the bug**
CSS is "buggy" and inconsistent.

### PR DESCRIPTION
**Pull request by QvistJ**
Closes #24

### CODE REVIEW

**Comment by QvistJ** [comment]
Installed WIP

**Comment by QvistJ** [comment]
The profile is still a bit weird, the name, img and role does not align properly.

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -14,151 +14,155 @@
...
-      margin-top: 0px;
-    }
-  }
-}
 
-.gravatar {
-  float: left;
-  margin-right: 10px;
-}
+// aside {
```
Comment (MSaurus) [inline app/assets/stylesheets/application.scss]: If you don't need it then you can remove it and save memory :D

File: app/views/microposts/_micropost.html.erb [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+  </div>
+  <div class="card-body">
+    <blockquote class="blockquote mb-0">
+      <p class="card-text"><%= micropost.content %></p>
+      <%= image_tag micropost.picture.url if micropost.picture? %>
+      <footer class="blockquote-footer"> 
+        Posted <%= time_ago_in_words(micropost.created_at) %> ago.
+        <% if current_user?(micropost.user) %>
+          <%= link_to "Delete", micropost, method: :delete,
+                                          data: { confirm: "You sure?" } %>
```
Comment (MSaurus) [inline app/views/microposts/_micropost.html.erb]: This line does not look like it's indent properly?

**Pull request merged**